App.tsx

```diff
@@ -1,20 +1,52 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from "react";
+import { StyleSheet, View, FlatList } from "react-native";
+import TodoItem from "./components/TodoItem";
+import TodoInput from "./components/TodoInput";
 
 export default function App() {
+  const [todosList, setTodosList] = useState([]);
+
+  function addTodoText(enteredTodoText) {
+    if (enteredTodoText != "") {
+      setTodosList([
+        ...todosList,
+        { text: enteredTodoText, id: Math.random().toString() },
+      ]);
+    }
+  }
+
   return (
     <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
+      <TodoInput onTodoAdded={addTodoText} />
+
+      <FlatList
```

Comment:
Instead of `<FlatList>` please use `<ScrollView>`. This will help you learn more. FlatList is a performance improvement and advanced component.

Reply:
Actually first use `<View>` and then after that, we will see after we add more than 10 or so items, we need to scorll. Then at that time please move to `<ScrollView>`

Reply:
Please Review the code . I Make List in View tag